src/mscorlib/src/System/Runtime/InteropServices/WindowsRuntime/WindowsRuntimeMarshal.cs

```diff
@@ -360,13 +360,10 @@ public override string ToString()
                 {
                     return "(" + target + ", " + method + ")";
                 }
-            }
 
-            internal class EventCacheKeyEqualityComparer : IEqualityComparer<EventCacheKey>
-            {
-                public bool Equals(EventCacheKey lhs, EventCacheKey rhs)
+                public bool Equals(EventCacheKey other)
                 {
-                    return (Object.Equals(lhs.target, rhs.target) && Object.Equals(lhs.method, rhs.method));
+                    return (ReferenceEquals(target, other.target) && ReferenceEquals(method, other.method));
```

Comment:
Why is this safe to change this to ReferenceEquals?

Reply:
Bad habit :-/

Reply:
Fixed it